### ISSUE TITLE
Send digest mail added to queue even when turned off

### ISSUE THREAD

**Issue by bartdigitalpulse** [solspace/craft3-freeform#1024]
**Describe the bug or issue you're experiencing**

I noticed the "Send digest mail" queue task getting randomly added to the queue. No e-mails are filled in. The mails are not getting send but seems useless to trigger the queue when no e-mails are filled in.

Settings in project.yaml:

```
alertNotificationRecipients: null
allowFileTemplateEdit: true
autoScroll: true
autoScrollToErrors: '1'
badgeType: all
blockedEmails: ''
blockedEmailsError: 'Invalid Email Address'
blockedIpAddresses: ''
blockedKeywords: ''
blockedKeywordsError: 'Invalid Entry Data'
bypassSpamCheckOnLoggedInUsers: ''
clientDigestFrequency: 1
clientDigestRecipients: null  
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 4.7.1
- Freeform Version: 5.0.7
- Freeform Edition: Pro
- Fresh Install or Upgrade:
Upgrade

**Comment by kjmartens** [comment]
Hi @bartdigitalpulse,

Thanks for reporting this. We'll likely disable this job from being sent to the queue when no email addresses are configured for the feature. 🙂

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.0.10+. 🙂
